### PR TITLE
[Refactor] Added uploadImport and ViewSwitch

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: Yash Kamboj <yashkamboj29@gmail.com>

**Description** Added uploadImport and ViewSwitch which is used in applications , filters , designs and performance page
Added undeploy icon
renamed prefrences file name

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/UploadImport.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  const handleOpenModal = () => setOpen(true);
+  const handleCloseModal = () => setOpen(false);
+  
+  return (
+    <>
+    <Button aria-label="URL-Upload" data-cy="import-button" variant="contained"
+          color="primary"
+          size="large"
+          onClick={handleOpenModal} sx={{marginBottom: theme.spacing(2) }} >
+          <LinkIcon style={{ paddingRight : ".35rem" }} />
```
Comment (Chadha93) [inline ui/components/UploadImport.js:21]: Use `sx` prop here to style.

File: ui/components/UploadImport.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        <>
+       <Grid container sx={{paddingTop: theme.spacing(2)}} >
+        <Grid item xs={3}>
+          <h4 > FROM URL </h4>
+        </Grid>
+        <Grid item xs={9}>
+          <TextField
+            variant="outlined"
+            label={`URL for ${configuration}`}
+            style={{ width : "100%" }}
```
Comment (Chadha93) [inline ui/components/UploadImport.js:38]: same

File: ui/components/UploadImport.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      <hr />
+      <Grid container spacing={2}>
+        <Grid item xs={3}>
+          <h4> UPLOAD FILE </h4>
+        </Grid>
+        <Grid item xs={6}>
+          <TextField
+            variant="outlined"
+            label="Filename"
+            sx={{ width : "100%" }}
```
Comment (Chadha93) [inline ui/components/UploadImport.js:51]: good

File: ui/components/ViewSwitch.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { ToggleButton, ToggleButtonGroup, } from "@mui/material";
+import GridOnIcon from '@mui/icons-material/GridOn';
+import TableChartIcon from "@mui/icons-material/TableChart";
+
+function ViewSwitch({ view, changeView }) {
+  console.log(view)
```
Comment (Chadha93) [inline ui/components/ViewSwitch.js:6]:
```suggestion
```

**Review by Chadha93 — COMMENTED** [review]
@YashKamboj Good work, some minors to look at.

**Comment by YashKamboj** [comment]
@Chadha93 Made all the requested changes

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**